ROADMAP.md

```diff
@@ -1,3 +1,28 @@
+1. Before the community meeting, compile feedback from the GitHub discussion into GitHub issues to track individual action items. Issues should be placed on the `wasmCloud Roadmap` project with the work stage of `Now`, applicable roadmap category of `Documenatation`, `New Feature`, or `Improvement`, and the applicable quarter e.g. `Q4 2024`.
+1. In the [wasmCloud Roadmap GitHub project](https://github.com/orgs/wasmCloud/projects/7), create a `New View` that selects each issue from the current quarter, and name the roadmap `QX XXXX` e.g. `Q4 2024`.
+1. During the community meeting, review created issues from the discussion. At least 15 minutes of the community meeting should be left available for additional discussion for items that did not make it onto the GitHub discussion.
+1. After the community meeting, finalize the Roadmap by outlining the high level initiatives and the GitHub project on the [wasmCloud Roadmap](https://wasmcloud.com/docs/roadmap/) page, which serves as a place that onlookers can see the outcomes of the planning session.
+
+## What goes on the Roadmap?
+
+wasmCloud roadmaps are a projection of the communities priorities, requests and feedback. Only issues that are completable in the three month period should go on the roadmap, and if they are longer efforts should be broken down into smaller increments. No issue is too small to go on the wasmCloud roadmap. There is no limit to the amount of tasks that can be put on the roadmap, but each quarterly planning session should review if too much was put on the previous roadmap.
+
+Roadmap features are generally divided into the following categories:
+1. Documentation: Issues that improve documentation either on the [website](https://wasmcloud.com) or in READMEs around the project
+1. New Feature: Features that will provide additional functionality in wasmCloud, wasmCloud SDKs or CLIs.
+1. Improvement: Fixes for existing bugs, developer experience rough edges, or rounding out an existing feature
+
+Quarterly roadmaps are not committments to completing _all_ work on the roadmap, but they are indicative of the priorities of the project. Features that are requested from wasmCloud community members and maintainers that are categorizable as `good-first-issue`s (in that they can be completed by developers new to the project) are highly encouraged, as are features that can be completed by the individual proposing them.
+
+If you have any concerns with roadmap planning or the process, please reach out to any of the [wasmCloud organization maintainers](https://github.com/wasmCloud/wasmCloud/blob/main/MAINTAINERS.md?plain=1#L7) or send a message in [Slack](https://slack.wasmcloud.com). If you are a wasmCloud  maintainer and you'd like to lead a roadmapping session, please reach out to the organization maintainers.
```

Comment:
Normally wouldn't matter, but since it is docs, can you remove the extra space in between `wasmCloud` and `maintainer`?